apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/spi/springsecurity/SpringSecurityUserService.java

```diff
@@ -75,21 +75,25 @@ public void createOrUpdate(UserPO user) {
       managedUser.setPassword(newPassword);
       managedUser.setEmail(user.getEmail());
       managedUser.setUserDisplayName(user.getUserDisplayName());
+      managedUser.setEnabled(user.getEnabled());
       userRepository.save(managedUser);
     }
   }
 
   @Override
-  public List<UserInfo> searchUsers(String keyword, int offset, int limit) {
-    List<UserPO> users = this.findUsers(keyword);
+  public List<UserInfo> searchUsers(String keyword, int offset, int limit, boolean includeInactiveUsers) {
+    List<UserPO> users = this.findUsers(keyword, includeInactiveUsers);
     if (CollectionUtils.isEmpty(users)) {
       return Collections.emptyList();
     }
     return users.stream().map(UserPO::toUserInfo)
         .collect(Collectors.toList());
   }
 
-  private List<UserPO> findUsers(String keyword) {
+  private List<UserPO> findUsers(String keyword, boolean includeInactiveUsers) {
+    if(includeInactiveUsers){
```

Comment:
We also need to implement the logic to search users by keyword when includeInactiveUsers is true.